Buildings/Experimental/DHC/Networks/Connections/Examples/Connection1PipeExample.mo

```diff
@@ -0,0 +1,150 @@
+within Buildings.Experimental.DHC.Networks.Connections.Examples;
```

Comment:
In Buildings.Experimental.DHC.Networks.Connections.Examples.Connection1PipeExample and Buildings.Experimental.DHC.Networks.Connections.Examples.Connection2PipeExample, the scale of bottom subplot is not right.

Reply:
Adjusted scale of plonts in .mos scripts.

Reply:
@EttoreZ Same needs to be done for ``Connection2PipeExample.mo``.